src/components/SQForm/SQFormDatePicker.tsx

```diff
@@ -29,7 +60,7 @@ function SQFormDatePicker({
   muiFieldProps = {},
   muiTextInputProps = {},
   isCalendarOnly = false
-}) {
+}: SQFormDatePickerProps): JSX.Element {
```

Comment:
I don't think it matters too much but I believe we prefer `React.ReactElement` over `JSX.Element`

Reply:
We have used `JSX.Element` in the SQForm TypeScript version.